### ISSUE TITLE
Multi ContolNet Network Selection

### ISSUE THREAD

**Issue by Googolplexed06** [jexom/sd-webui-depth-lib#9]
ControlNet supports running multiple networks with different images on a single generation through the Multi ControlNet setting. Currently Depth Library's "Send to ControlNet" button sends the image to ControlNet network 0 exclusively. It would be ideal to specify which network number (0 up to 9 I believe) to send the pose to. This specification is present in fkunn1326's OpenPose Editor [https://github.com/fkunn1326/openpose-editor](https://github.com/hnmr293/posex/issues/url) for reference.

**Comment by jexom** [comment]
Yeah, thought about it, the problem is that it doesn't actually work in the OpenPose editor. Neither for me nor for the people I spoke to about it.

**Comment by wywywywy** [comment]
To get it to work, you have to have the right Network tab already open.

I tested it to automate switching to right tab but so far it's not been reliable. I'll give it another go and see if I can get it to work.
